recipe/meta.yaml

```diff
@@ -1,60 +1,110 @@
-{% set name = "re2" %}
-{% set version = "2023-03-01" %}
+{% set version = "2023-06-02" %}
 {% set dotversion = version.replace('-', '.') %}
-# switching from C++14 to C++17 unfortunately introduces a different ABI, see #61;
-# create a fake version bump so we can migrate the new version safely
-{% set abi_bump_version = dotversion.replace('.01', '.02') %}
+{% set build_num = 0 %}
```

Comment:
This will break the bot's auto-increment code.
```suggestion
```

Reply:
Pretty sure the bot handles that these days, but I'm fine with changing it back (had forgotten about `PKG_BUILDNUM`...)